airbyte-webapp/src/views/Connector/ServiceForm/components/Sections/auth/useOauthFlowAdapter.tsx

```diff
@@ -38,13 +41,16 @@ function useFormikOauthAdapter(connector: ConnectorDefinitionSpecification): {
     }
 
     setValues(newValues);
+    setHasRun(true);
   };
 
   const { run, loading, done } = useRunOauthFlow(connector, onDone);
+  const connectionObjectEmpty = !!Object.keys((values?.connectionConfiguration as object) ?? {}).length;
```

Comment:
Wouldn't that simply check if there is any configuration parameter set, i.e. this could be also any other parameter of the form being set?

Reply:
![Screen Shot 2022-08-12 at 1 37 59 PM](https://user-images.githubusercontent.com/4967694/184413431-b39cf5cc-2f17-43f5-9ff5-e5d78dd7ef9b.png)
The oauth interfaces that I interacted with were all like this, so it seemed a safe call. If that's not the case than this may be a good bit more complicated.